### PR TITLE
Lusas_Toolkit: Issue #178 - Switching to NuGet packages for external references

### LINKED ISSUE #178: Lusas_Toolkit: Use NuGet to target external dlls
#### Description:
There should be no direct reference to local dlls anymore and only reference through NuGet. This is necessary to enable CI through AppVeyor.

### PR DESCRIPTION
**Pull request by adecler**
### Issues addressed by this PR

Closes #178

### Test files
Just make sure everything is still working

### CODE REVIEW

File: Lusas_Adapter/Lusas_Adapter.csproj [hunk 1]
```diff
@@ -73,10 +73,9 @@
     <Reference Include="Geometry_oM">
       <HintPath>..\..\BHoM\Build\Geometry_oM.dll</HintPath>
     </Reference>
-    <Reference Include="Lusas.Interop, Version=15.2.1601.22607, Culture=neutral, PublicKeyToken=ee620220e102840a, processorArchitecture=x86">
-      <SpecificVersion>False</SpecificVersion>
+    <Reference Include="Lusas.Interop, Version=17.0.1601.28524, Culture=neutral, PublicKeyToken=ee620220e102840a, processorArchitecture=x86">
+      <HintPath>..\packages\BHoM.Interop.Lusas170.1.0.0\lib\net45\Lusas.Interop.dll</HintPath>
```
Comment (peterjamesnugent) [inline Lusas_Adapter/Lusas_Adapter.csproj]: It does not appear to be able to push anything and gives the following message:

> 1. This component failed to run properly. Are you sure you have the correct type of inputs?
> Check their description for more details. Here is the error provided by the method:
> A null reference pointer was passed to the stub. (Exception from HRESULT: 0x800706F4)

Comment (peterjamesnugent) [inline Lusas_Adapter/Lusas_Adapter.csproj]: It does not appear to be able to push anything and gives the following message:

> 1. This component failed to run properly. Are you sure you have the correct type of inputs?
> Check their description for more details. Here is the error provided by the method:
> A null reference pointer was passed to the stub. (Exception from HRESULT: 0x800706F4)

Comment (adecler) [reply Lusas_Adapter/Lusas_Adapter.csproj]: Can you check the version you have for Lusas ?

image

Comment (peterjamesnugent) [reply Lusas_Adapter/Lusas_Adapter.csproj]: Apologies for just getting around to this - version shown below.

image

**Comment by FraserGreenroyd** [comment]
Discussed in BHoM call, @FraserGreenroyd to take a look at this to assist closing out this PR.

**Comment by FraserGreenroyd** [comment]
@adecler apologies I did not get chance to review this today - however, does @peterjamesnugent comment from this morning provide any further assistance in understanding the issue? If not I'll try and get this looked at on Monday :smile:

**Comment by peterjamesnugent** [comment]
@adecler @FraserGreenroyd I have copied the LusasInterop.dll here:
https://burohappold.sharepoint.com/:f:/s/BHoM/EqFQWoHnK-FIguDkupxPnvQBLz5MMNnX5lIWw5ARhiflRQ?e=Arn5df

I have also the copied over the Programs folder where it is stored.

**Comment by adecler** [comment]
@peterjamesnugent , I have now created the new NuGet package with what you provided above.
I have updated the link in the code. Can you confirm it now works for you ?

**Comment by al-fisher** [comment]
Fantastic - thanks both!

**Pull request merged**